core/src/services/mongodb/backend.rs

```diff
@@ -0,0 +1,315 @@
+use async_trait::async_trait;
+use mongodb::bson::Binary;
+use mongodb::bson::{doc, Document};
+use mongodb::options::{ClientOptions, UpdateOptions};
+use std::fmt::{Debug, Formatter};
+use std::sync::Arc;
+use tokio::sync::OnceCell;
+
+use crate::raw::adapters::kv;
+use crate::*;
+
+#[derive(Default)]
+pub struct MongoDbBuilder {
```

Comment:
Please use `MongodbBuilder` since it's scheme is `mongodb` instead of `mongo-db`.